src/App.tsx

```diff
@@ -1,24 +1,178 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './Components/TodoList';
+import { TodoFilter } from './Components/TodoFilter';
+import { FilterType } from './types/FilterType';
+import { ErrorMessage } from './Components/ErrorMessage';
 
-const USER_ID = 0;
+const USER_ID = 10390;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(FilterType.All);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [title, setTitle] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const completedTodo = todos.filter(todo => todo.completed === true);
+  const activeTodos = todos.length - completedTodo.length;
+
+  const getFiltered = (filter: FilterType) => {
```

Comment:
just wrap with `useMemo` 
```suggestion
  const filteredTodo = useMemo((filter: FilterType) => {
```